From 5aaabc9e76ea2053489371b04c596a0f87b9ad17 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 15 Feb 2024 17:19:28 +0100
Subject: [PATCH 0599/1695] dts: stm32: add wake up parent for pmic2 on
 stm32mp257f-ev1 board

Reference PWR as wakeup-parent for PMIC on this board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4dd8bf89e9b9fccb2aa8360c702c7c465fa9b373 dts: stm32: add wake up parent for pmic2 on stm32mp257f-ev1 board]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I70e900894357823883178747d28ff435aa348f18
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 62736d408..6a9627f2f 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -63,6 +63,7 @@
 		compatible = "st,stpmic2";
 		reg = <0x33>;
 		st,wakeup-pin-number = <1>;
+		wakeup-parent = <&pwr>;
 		status = "okay";
 
 		regulators {
-- 
2.47.2

